tests/Database/DatabaseEloquentCollectionQueueableTest.php

```diff
@@ -5,19 +5,19 @@
 use Illuminate\Database\Eloquent\Collection;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\Pivot;
-use Mockery;
+use Mockery as m;
```

Comment:
I think it is not clear enough to just use "m" as an alias. I had to look at to the top to understand what is "m"

Reply:
`m` is used everywhere in the tests; those were the only 11 files where it was not the case.
And within the tests `m::mock` are pretty self-explanatory (imo)

Reply:
`m` is a standard across PHP projects to alias Mockery.